platform/brains2/src/duo/tempo.cpp

```diff
@@ -1,4 +1,4 @@
-#include "lib/tempo.h"
+#include "tempo.h"
```

Comment:
```
fatal error: tempo.h: No such file or directory
    1 | #include "tempo.h"
      |          ^~~~~~~~~
```

Reply:
on my machine tempo.h cannot be found if it is not prefixed by lib/ . Any idea on why this is the case?

Reply:
Mistake from my side. I fixed it in the last commit.
I believe I was moving tempo.h/cpp from `lib` to the `duo` source, since it's support for the main firmware rather than the general `lib` (which is for the whole board, and hence can't reference anything in `firmware`). With `tempo.cpp` in `lib`, we currently can't build the demos.
I will fix this in `main` instead though.